frontend/src/features/apartment/ApartmentSalesPage.tsx

```diff
@@ -202,26 +309,32 @@ const LoadedApartmentSalesPage = ({apartment}: {apartment: IApartmentDetails}) =
                             <Button
                                 theme="black"
                                 onClick={handleCalculateButton}
+                                /*
+                                disabled={
+                                    !(formData.purchase_date && formData.apartment_share_of_housing_company_loans)
+                                }
+                                */
```

Comment:
Is there plans to do something with this later?
If so, add TODO comment to mark it so we remember to do it and make it clear why it's commented out
If not, this can be removed :)

Reply:
Yep! This is left out due to me wanting to apply the "proper" front end form-validation to this, instead of having custom stuff.. Also it's was left out to demonstrate the hitasToast highlights (which were only implemented in this empty error message for now), as you couldn't get to the notification (== push the button) if the fields weren't empty.